File: TGAPEngine/src/DrawingContext.cpp
#include "DrawingContext.h"

namespace engine{
	DrawingContext::DrawingContext(Camera * camera, Light * lights[]) :activeCamera(camera)
	{
		lightsNumber = 0;
		for (int i = 0; i < 4; i++){
			if (lights[i] != NULL){
				activeLights[i] = lights[i];
				lightsNumber++;
			}
			else{
				break;
			}

		}
	}


	DrawingContext::~DrawingContext()
	{
	}
}

File: CS/TGAPE_CS/Engine/Renderer/DrawingContext.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TGAPE_CS.Engine.Renderer
{
    public class DrawingContext
    {
        public Camera ActiveCamera;
        public List<Light> ActiveLights;

        public DrawingContext(Camera camera, List<Light> lights)
        {
            ActiveCamera = camera;
            ActiveLights = lights;
        }
    }
}
